proto/ddl_service.proto

```diff
@@ -15,8 +15,7 @@ message CreateDatabaseRequest {
 
 message CreateDatabaseResponse {
   common.Status status = 1;
-  uint32 database_id = 2;
```

Comment:
No need to return the id for all create ddl, they are never used. And it doesn't have any compatible issues, since they are never persisted.